.gitlab-ci.yml

```diff
@@ -202,16 +196,22 @@ test:prep:
 
 test:staging-deployment:chrome:
   extends: .template:test:staging-deployment
+  variables:
+    TEST_SAML_CREDENTIALS: "chrome"
   script:
     - npm run test-ci
 
 test:staging-deployment:firefox:
   extends: .template:test:staging-deployment
+  variables:
+    TEST_SAML_CREDENTIALS: "firefox"
   script:
     - npm run test-ci -- --browser=firefox
 
 test:staging-deployment:webkit:
   extends: .template:test:staging-deployment
+  variables:
+    TEST_SAML_CREDENTIALS: "webkit"
   script:
```

Comment:
These values are already provided OOTB by playwright, please use them. (for examples you can search the codebase for `webkit` or check [here](https://github.com/mendersoftware/gui/blob/master/tests/e2e_tests/integration/05-deviceDetails.spec.ts#L40))

Reply:
aha! I knew they must be there -- forgot to check, sorry for that.